dashboard/app/models/frequent_unsuccessful_level_source.rb

```diff
@@ -30,15 +30,15 @@ def self.populate(freq_cutoff, game_name)
     # was specified, limit the query to the named game.
     game_id = game_name && Game.where('name = ?', game_name).first.id
     if game_id
-      query = "select a.level_source_id, a.level_id, count(*) as num_of_attempts
+      query = "select a.level_source_id, max(a.level_id), count(*) as num_of_attempts
```

Comment:
This and the below change are here to support MySQL 5.7 defaults. Specifically, a change regarding the default [handling of GROUP BY statements](https://dev.mysql.com/doc/refman/5.7/en/group-by-handling.html) requires this ambiguous query to be made more explicit.